assistant/src/main/java/com/ibm/watson/assistant/v1/model/UpdateDialogNodeOptions.java

```diff
@@ -121,6 +121,7 @@
   private String newDigressOut;
   private String newDigressOutSlots;
   private String newUserLabel;
+  private Boolean newDisambiguationOptOut;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/watson-developer-cloud/java-sdk/pullRequest?prid=4552601)